src/hammer-vlsi/generate_properties.py

```diff
@@ -119,7 +119,11 @@ def main(args) -> int:
                                         InterfaceVar("output_files", "List[str]",
                                                      "output collection of mapped (post-synthesis) RTL files"),
                                         InterfaceVar("output_sdc", "str",
-                                                     "(optional) output post-synthesis SDC constraints file")
+                                                     "(optional) output post-synthesis SDC constraints file"),
+                                        InterfaceVar("output_access", "str",
+                                                     "output access file for gate level simulation"),
+                                        InterfaceVar("output_force_regs", "str",
+                                                     "output force regs tcl file for gate level simulation register initialization")
                                         # TODO: model CAD junk
```

Comment:
this is missing a few outputs we probably want: SAIF for activity factors (maybe call it `activity_file`, VCD/VPD (maybe call it `waveform_file`)

Reply:
I'm not sure that we need to pass the VCD/VPD files downstream (feel free to correct me).  I agree though about adding some outputs.  We will need to pass SDF files and SPF files for different sims.

Reply:
The VCD files are consumed by the dynamic power analysis tools. We can add them later.